backend/src/external/aws/lambdas/text_extractor.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
```

Comment:
Shall I also include adding logging over our print statements? I thought we had a story for this but I don't mind the approach of refactoring as we go through testing either.

Reply:
If you have the presence of mind to convert to `logging`, yes, please do it.